spec/integration/helpers/start-atom.js

```diff
@@ -80,7 +70,6 @@ const buildAtomClient = async (args, env) => {
           'dev',
           'safe',
           `user-data-dir=${userDataDir}`,
```

Comment:
[The linter pointed this out](https://dev.azure.com/DeeDeeG/b/_build/results?buildId=935&view=logs&j=3a559e2a-952e-58d2-b8db-2e604a9266d7&t=6f845a59-a86d-5907-2b45-02c2fb4984eb&l=14): No comma needed, because this is the last item in the list.

Other than this failing linting due to the extra comma, CI passes if applying these changes on top of `master`:

https://dev.azure.com/DeeDeeG/b/_build/results?buildId=935